### DISCUSSION TITLE
Accès difficiles aux dossiers Android data net.Osmand.Plus Track dossiers Import et Rec

### DISCUSSION THREAD

**Discussion by Goupil30** [community/community#120911] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Bonjour,
Osmand proposition, déplacement dossiers Traces 
Depuis Android 13 et 14 il devient difficile à causes des restrictions d'accès de Google pour accéder aux dossiers contenus dans Android data net.Osmand.Plus Track dossiers Import et Rec
Merci de voir s'il serait possible de déplacer à volonté ces dossiers Tracks et son contenu vers des dossiers facilement accessibles par l'utilisateur.
Bien cordialement
JM Luitaud

**Comment by ebndev** [comment]
Hi @Goupil30, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
